src/corelibs/U2View/src/ov_msa/MSAEditor.h

```diff
@@ -39,7 +39,7 @@ class PairwiseAlignmentWidgetsSettings {
     PairwiseAlignmentWidgetsSettings()
         : firstSequenceId(U2MsaRow::INVALID_ROW_ID),
           secondSequenceId(U2MsaRow::INVALID_ROW_ID), inNewWindow(true),
-          pairwiseAlignmentTask(NULL), showSequenceWidget(true), showAlgorithmWidget(false),
+          pairwiseAlignmentTask(), showSequenceWidget(true), showAlgorithmWidget(false),
```

Comment:
You don't need `pairwiseAlignmentTask` here at all, cuz QPointer has the default constructor